Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -168,6 +211,29 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
     model.internalPolyData.modified();
     outData[0] = model.internalPolyData;
   };
+
+  publicAPI.updateActorVisibility = (
+    renderingType = RenderingTypes.FRONT_BUFFER,
+    widgetVisible = true,
+    ctxVisible = true,
+    handleVisible = false
+  ) => {
+    superClass.updateActorVisibility(
+      renderingType,
+      widgetVisible,
+      ctxVisible,
+      handleVisible
+    );
+    if (model.fromLineWidget) {
+      const visibility = model.handleVisibility;
```

Comment:
use the existing behavior of `visibilityFlagArray` (see WidgetRepresentation)

Reply:
incredible! makes the whole code way more readable and avoids to overload updateActorVisibility!